dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite."
+DESCRIPTION="It is for users and developers wanting to create databases, search, and edit data. It uses a familiar spreadsheet-like interface, and you don't need to learn complicated SQL commands.\
```

Comment:
Can you please change DESCRIPTION to:
```
DESCRIPTION="DB Browser for SQLite is a high quality, visual, open source tool \
to create, design, and edit database files compatible with SQLite."
```
(I think we don't need a longer description but you can keep it if you prefer, as long as you follow our [HaikuPorter guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering), of course.)